src/components/Author/index.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react"
+import { Link } from "gatsby"
+import dateFormat from "dateformat"
+import PropTypes from "prop-types"
+
+import Image from "../../images/team/placeholder-v.jpg"
+
+import * as styles from "./index.module.scss"
+
+const renderAuthor = ({ key, name }) => (
+  <Link to={`/blog/author/${key}`}>{name}</Link>
+)
```

Comment:
I believe @gabrielpoca converted all the arrow functions

```suggestion
function renderAuthor({ key, name })  {
 return <Link to={`/blog/author/${key}`}>{name}</Link>
}
```